src/main/js/dynamic/loaded/users/UserTable.js

```diff
@@ -0,0 +1,121 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types'
+import TableDisplay from 'field/TableDisplay';
+import TextInput from 'field/input/TextInput';
+import PasswordInput from 'field/input/PasswordInput';
+import CheckboxInput from 'field/input/CheckboxInput';
+import { connect } from 'react-redux';
+import { createNewUser, deleteUser, fetchUsers } from 'store/actions/users';
+
+class UserTable extends Component {
+    constructor(props) {
+        super(props);
+
+        this.retrieveData = this.retrieveData.bind(this);
+        this.createColumns = this.createColumns.bind(this);
+        this.handleChange = this.handleChange.bind(this);
+        this.onSave = this.onSave.bind(this);
+
+        this.state = {};
+    }
+
+    componentDidMount() {
+        this.retrieveData();
+    }
+
+    createColumns() {
+        return [
+            {
+                header: 'username',
+                headerLabel: 'Username',
+                isKey: true
+            },
+            {
+                header: 'email_address',
+                headerLabel: 'Email',
+                isKey: false
+            },
+            {
```

Comment:
Have to double check here but we may only want username and email address shown.  The expired, locked, and password_expired fields are there because that is the type of information the user models can support.  However in Alert right now we don't have any of those features to expire a user or their password to my knowledge.  We may need to implement that.

Reply:
What about the enabled field? Will this be hidden as well?